ibis/backends/duckdb/compiler.py

```diff
@@ -400,3 +400,22 @@ def visit_HexDigest(self, op, *, arg, how):
 
     def visit_StringConcat(self, op, *, arg):
         return reduce(lambda x, y: sge.DPipe(this=x, expression=y), arg)
+
+    def visit_Enumerate(self, op, *, arg, start: int):
+        # adapted from visit_ArrayZip
+        i = sg.to_identifier("i")
```

Comment:
This implementation isn't correct, it's not enumerating globally when it should be.